agent_runtime_profile/CLAUDE.narration.md

```
@@ -23,6 +23,9 @@
 ### 音频规范
 - **BGM 自动禁止**：生成端已在视频 prompt 末尾自动追加「禁止出现：BGM、文字字幕、水印」，无需手动追加，video_prompt 里也不要描述 BGM / 配乐
 
+### 视频 prompt 措辞
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)